llvm/docs/CommandGuide/llvm-objcopy.rst

```diff
@@ -78,10 +78,59 @@ multiple file formats.
  Enable deterministic mode when copying archives, i.e. use 0 for archive member
  header UIDs, GIDs and timestamp fields. On by default.
 
+.. option:: --globalize-symbol <symbol>
+
+ Mark any defined symbols named ``<symbol>`` as global symbols in the output.
+ Can be specified multiple times to mark multiple symbols.
+
+ This option is only supported for ELF and MachO.
```

Comment:
FWIW, I don't think we have this note on other options that are specific to a subset of the supported formats we have, so I'd omit it to avoid confusion.

Reply:
Removed.